slackbot/manager.py

```diff
@@ -5,15 +5,14 @@
 from glob import glob
 from six import PY2
 from importlib import import_module
-from slackbot import settings
 from slackbot.utils import to_utf8
 
 logger = logging.getLogger(__name__)
 
 
 class PluginsManager(object):
-    def __init__(self):
-        pass
+    def __init__(self, settings):
+        self.settings = settings
```

Comment:
See above comment, this might not be needed if we are only running one bot out of this code. Multiple bots will run in different processes, each process can be started with different env vars